dask_cuda/tests/test_device_host_file.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import cupy
+from dask_cuda.device_host_file import DeviceHostFile
+from random import randint
+
+import pytest
+from dask_cuda.utils_test import gen_random_key
+from cupy.testing import assert_array_equal
+
+
+@pytest.mark.parametrize("num_host_arrays", [1, 10, 100])
+@pytest.mark.parametrize("num_device_arrays", [1, 10, 100])
+@pytest.mark.parametrize("array_size_range", [(1, 1000), (100, 100), (1000, 1000)])
+def test_device_host_file_short(num_device_arrays, num_host_arrays, array_size_range):
+    import tempfile
+
+    tmpdir = tempfile.TemporaryDirectory()
+    dhf = DeviceHostFile(
+        device_memory_limit=1024 * 16, memory_limit=1024 * 16, local_dir=tmpdir.name
+    )
+
+    host = [
+        (gen_random_key(64), np.random.random(randint(*array_size_range)))
```

Comment:
Instead of `gen_random_key` would `"x-%d" % i` work?  That might help if we need to debug in the future, and it also removes the need for future readers to inspect and learn about the `gen_random_key` function.

Reply:
It would, I just wanted to have some long names, that are closer in complexity to those of real arrays. If you'd still prefer to change it to the way you mention, just let me know.

Reply:
I'd prefer the simpler names.  This desire comes from having to dive through code like this years after it has been written to help users debug failed tests.  The fewer functions I need to read and files I need to look through, the better :)

Reply:
Also, if you're looking for a random string in Python in the future, I recommend the `uuid` module

```python
In [1]: import uuid

In [2]: uuid.uuid4().hex
Out[2]: '2bffbb54625d4aa195f1993d3809ff5c'
```

Reply:
I thought about that too, but I wanted to have some special characters too, as I've seen is the case in real usage. But I'll simplify this as per your suggestion.

Reply:
I appreciate it!  Sorry for the nitpicks.